app/src/main/java/com/example/teamyuml/decentworkmobile/views/SplashScreen.java

```diff
@@ -14,7 +14,7 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        startActivity(new Intent(SplashScreen.this, NoticeList.class));
+        startActivity(new Intent(SplashScreen.this, Login.class));
```

Comment:
For my knowlegde of this project We start with NoticeList Activity rather than Login